packages/ui-app/src/Params/Param/Balance.tsx

```diff
@@ -0,0 +1,177 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { ApiProps } from '@polkadot/ui-react-rx/types';
+import { I18nProps } from '../../types';
+import { Props as BareProps } from '../types';
+
+import BN from 'bn.js';
+import React from 'react';
+import withApi from '@polkadot/ui-react-rx/with/api';
+
+import isValidBalance from '../../util/isValidBalance';
+import { defaultMaxLength } from '../../util/chainSpec';
+import { keydown, keyup } from '../../util/keyboard';
+import InputNumber from '../../InputNumber';
+import translate from '../../translate';
+import { KEYS_ALLOWED, KEYS_PRE } from '../../constants';
+import Bare from './Bare';
+
+type Props = I18nProps & ApiProps & BareProps;
+
+type State = {
+  error?: React.ReactNode,
+  info?: React.ReactNode,
+  warn?: React.ReactNode,
+  isPreKeyDown: boolean
+};
+
+class Balance extends React.PureComponent<Props, State> {
+  state: State;
+
+  constructor (props: Props) {
+    super(props);
+
+    this.state = {
+      error: '',
+      warn: '',
+      info: '',
+      isPreKeyDown: false
+    };
+  }
+
+  render () {
+    const { className, defaultValue: { value }, isError, label, style, t, withLabel } = this.props;
+    const { error, info, warn } = this.state;
+    const defaultValue = new BN((value as BN).toString(10) || '0').toString(10);
+
+    return (
+      <Bare
+        className={className}
+        style={style}
+      >
+        <InputNumber
+          className='large'
+          defaultValue={defaultValue || '0'}
+          error={error}
+          info={info}
+          isError={isError}
+          label={label}
+          maxLength={defaultMaxLength}
+          onChange={this.onChange}
+          onKeyDown={this.onKeyDown}
+          onKeyUp={this.onKeyUp}
+          placeholder={t('account.balance.placeholder', {
+            defaultValue: 'Between 1 DOT and the available DOT balance of the account'
```

Comment:
Can be 0. (Additionally this now does not match up with the default shown above)